test/agrovoc-allegrograph_on_hold.py

```diff
@@ -277,39 +273,31 @@ def testSelectByGETinN3(self):
         ct = result.info()["content-type"]
         assert True in [one in ct for one in _SPARQL_SELECT_ASK_POSSIBLE], ct
         results = result.convert()
-        results.toxml()
-        self.assertEqual(results.__class__.__module__, "xml.dom.minidom")
-        self.assertEqual(results.__class__.__name__, "Document")
+        self.assertEqual(type(results), dict)
```

Comment:
also not sure why the result should be a dict here, I'm not that familiar with this library sadly, if you could clarfiy please do.

Reply:
8347df7659fa64bd98bf892426e6223942dc0856